### PR TITLE
LPS-106100 frontend-js-aui-web - Fix regression bug with jQuery removal

### PR DESCRIPTION
**Pull request by kresimir-coko**
Hey Chema, I've adapted the solution to support more than just a string parameter, and made some of the code more modern than in the initial PR

### CODE REVIEW

**Comment by jbalsas** [comment]
ci:test:relevant

**Comment by jbalsas** [comment]
ci:test

File: modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/util.js [hunk 1]
```diff
@@ -296,23 +296,23 @@
...
 
 				if (Array.isArray(name)) {
-					selector =
-						'input[name=' +
-						name.join('], input[name=') +
-						STR_RIGHT_SQUARE_BRACKET;
+					selector = `input[name=${name.join('], input[name=')}]`;
 				} else {
-					selector = 'input[name=' + name + STR_RIGHT_SQUARE_BRACKET;
+					selector = `input[name=${name}]`;
```
Comment (jbalsas) [inline modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/util.js]: Is this missing quotes now? I think also the one above? => `input[name='foo']`

Let's avoid any unnecessary changes unrelated to the task. Each of them, as trivial as they seem increases the risk of regressions for a very little gain. Without tests to protect us here, the less we change the better.

File: modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/util.js [hunk 2]
```diff
@@ -328,11 +328,9 @@
 				if (selectClassName) {
 					const selectItem = form.querySelector(selectClassName);
 
-					if (allBoxChecked) {
-						selectItem.classList.add('info');
-					} else {
-						selectItem.classList.remove('info');
-					}
+					allBoxChecked
```
Comment (jbalsas) [inline modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/util.js]: This is less risky, but also unnecessary... I would avoid these changes for now. I'm also not convinced this reads any better, and since it actually has a side effect, I would even argue that this might even hurt a bit 🤔

File: modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/util.js (hunk 1 above)
Comment (kresimir-coko) [reply modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/util.js]: Yea both of them seem to be missing quotes, I didn't add them because the old one didn't have them.
Sure, so basically discard all the "modernizations"?

Comment (jbalsas) [reply modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/util.js]: > Sure, so basically discard all the "modernizations"?

Without tests, I'd feel more comfortable yes. There's no reason to incur in unnecessary risks and make this harder on ourselves

Comment (kresimir-coko) [reply modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/util.js]: Sure, I'll do that then

**Comment by kresimir-coko** [comment]
Hey Chema, I've removed the modernization part of the commit, tested it still works on both occurences and force-pushed

**Comment by jbalsas** [comment]
Thanks! I'll double down on this asap!

**Comment by jbalsas** [comment]
ci:test:relevant

**Comment by jbalsas** [comment]
ci:test

**Comment by jbalsas** [comment]
Pull request submitted to https://github.com/brianchandotcom/liferay-portal/pull/82337. <br><br>:octocat: *Sent from [GH](http://nodegh.io).*
